plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block-calculate-shipping.spec.js

```diff
@@ -166,12 +175,21 @@ baseTest.describe( 'Cart Block Calculate Shipping', () => {
 		await addAProductToCart( page, product1Id );
 		await page.goto( cartBlockPage.slug );
 
+		await page
+			.locator( 'button', {
+				hasText: 'Enter address to check delivery options',
```

Comment:
Sorry about the nitpick, and I am not an English native speaker, but this feels weird without the correct article usage eg "Enter an address.." or "Enter the address.." 🤔
Just making sure nothing is lost before this enters the translation tumblers 😅

Reply:
@mikejolley and @opr What are your thoughts as native English speakers?

@wavvves I briefly checked Amazon, and they also leave the articles out:

<img width="265" alt="Screenshot 2024-06-12 at 17 15 26" src="https://github.com/woocommerce/woocommerce/assets/3323310/24a097e6-89cc-4d2c-a661-f0527bae0d13">

I'd say, this is a common pattern in UIs, but let's see what Mike and Thomas are saying.

Reply:
> I'd say, this is a common pattern in UIs, but let's see what Mike and Thomas are saying.

I wouldn't say it is a pattern, it's instructions vs actions (eg. buttons). Instructions tend to convey more clarity and be less direct. plus we had "an" before. Amazon also has an example of this:

<img width="477" alt="image" src="https://github.com/woocommerce/woocommerce/assets/17236129/d4e21784-8514-4ac1-9fc9-bc6e5e960e68">

The instructions "change your delivery address" vs "Change address" on the button

Reply:
This should be extracted into some form of guidelines—I think we should get design input on this too. Actually thinking about it I think we should (if its not already) document practices on case and "full stops" on UI elements also because I still see inconsistencies even with the new stuff.

cc @ralucaStan 

Personally, `Enter address to check delivery options` is fine with me. This isn't a full sentence and its shown within a button element. I'm assuming this came from the design team anyway right?

In your example above @wavvves I would look and compare to the button, not the paragraph before. `Change Address` vs `Change your address` or similar. Our element is also a clickable button.

Reply:
>This should be extracted into some form of guidelines—I think we should get design input on this too. Actually thinking about it I think we should (if its not already) document practices on case and "full stops" on UI elements also because I still see inconsistencies even with the new stuff.

Thank you for the suggestion, Mike!  I have asked for the design input at https://github.com/woocommerce/woocommerce/pull/47953#discussion_r1643050142.
I have also cross-linked your comment there. 

>Personally, Enter address to check delivery options is fine with me. This isn't a full sentence and its shown within a button element. I'm assuming this came from the design team anyway right?

Yes, this came from the design.